cmd/init.go

```diff
@@ -108,22 +108,32 @@ func initCommon(args []string) error {
 		initOptions.MemberCount = memberCount
 	}
 
-	initOptions.OrgNames = make([]string, 0, initOptions.MemberCount)
-	initOptions.NodeNames = make([]string, 0, initOptions.MemberCount)
+	orgNames := make([]string, initOptions.MemberCount)
+	nodeNames := make([]string, initOptions.MemberCount)
 	if promptNames {
 		for i := 0; i < initOptions.MemberCount; i++ {
 			name, _ := prompt(fmt.Sprintf("name for org %d: ", i), validateFFName)
-			initOptions.OrgNames = append(initOptions.OrgNames, name)
+			orgNames[i] = name
 			name, _ = prompt(fmt.Sprintf("name for node %d: ", i), validateFFName)
-			initOptions.NodeNames = append(initOptions.NodeNames, name)
+			nodeNames[i] = name
 		}
 	} else {
 		for i := 0; i < initOptions.MemberCount; i++ {
-			id := randomHexString(3)
-			initOptions.OrgNames = append(initOptions.OrgNames, fmt.Sprintf("org_%s", id))
-			initOptions.NodeNames = append(initOptions.NodeNames, fmt.Sprintf("node_%s", id))
+			randomName := randomHexString(3)
```

Comment:
Did we switch away from sequential org names by default? Kinda liked the predictability of org0, org1, etc.

Reply:
Yes, that change happened a few commits ago. Hasn't been in a release yet. It made it easier when connecting to remote Fabric networks. Because once `org0` had be registered as an identity, it was used up. So if you ran `ff reset` `ff start` you'd be stuck in a permanently broken state because `org0` was previously registered, but the Fabconnect instance that held the key and been deleted.

Reply:
Open to suggestions on if we want to make that configurable or not though. If you liked `org0`, `org1` etc, you can specify that in the command line now though :)